src/payload/orderPayloadBuilder.ts

```diff
@@ -0,0 +1,69 @@
+import OrderPayload from "./orderPayload";
+import { Sender } from "../models/sender";
+import { Recipient } from "../models/recipient";
+
+export default class OrderPayloadBuilder {
+    quotationId: string | undefined;
+
+    sender: Sender | undefined;
+
+    recipients: Recipient[] | undefined;
+
+    fleetOption?: string;
+
+    isPODEnabled?: boolean;
+
+    isRecipientSmsEnabled?: boolean;
+
+    paymentMethod?: string;
+
+    partner?: string;
+
+    static orderPayload(): OrderPayloadBuilder {
+        return new OrderPayloadBuilder();
+    }
+
+    withQuotationID(quotationID: string): OrderPayloadBuilder {
```

Comment:
We are using different syntax (`quotationId` & `quotationID`) for the same data. Let's align them (fyi in our public doc we mention `quotationId`)

Reply:
done